init/arch_specs/eessi_arch_x86.spec

```diff
@@ -1,7 +1,10 @@
 # x86_64 CPU architecture specifications
+# The overview at https://github.com/InstLatx64/InstLatX64_Misc/tree/4802ff02415ed1ecdefddfd540615a27b617349c/SIMD_Euler may be helpful in defining this list
```

Comment:
use https://github.com/InstLatx64/InstLatX64_Misc/tree/main/SIMD_Euler as link here?

Reply:
Done in https://github.com/EESSI/software-layer/pull/1084/commits/267cecb4e04e3c7affaf03b7a1bb259800e2b889.